CityOfDetroit/bloom
# 1444+5/listing finder q4+5

# Pull Request Template

## Issue Overview

This PR addresses #1444 and #1445. Also a small UI bug mentioned [here](https://github.com/CityOfDetroit/bloom/issues/1443#issuecomment-1276634477)


## Description

This PR adds in questions 4 and 5 of the listing finder. In order to more easily accommodate for custom subtitles and intuitive string naming, I expanded the Finder Question type. It also includes a small UI improvement along with a slight change to rental costs since section 8 acceptances wasn't filtering correctly.

## How Can This Be Tested/Reviewed?

This can be tested by clicking through the form and ensuring that the form saves and applies the selections entered whether you click previous on the disclaimer page or finish and see it reflected in the listings.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
